.travis.yml

```diff
@@ -89,13 +89,17 @@ install:
 script:
   - cd ${TRAVIS_BUILD_DIR}
   - if [[ $TRAVIS_OS_NAME == 'osx' ]]; then ulimit -S -n 2048; fi
+  - if [[ ${TRAVIS_PULL_REQUEST} == "true" && \
+          ${BUILD_DOCS} == "true"]]; then \
```

Comment:
Syntax error: you need a space between `"true"` and ``]]``.

Travis completely fails on this at the moment.